theurgy.lic

```diff
@@ -62,6 +62,69 @@ def clean_altar
   put_in_container($settings.water_holder)
 end
 
+def prayer_bead
+  return unless DRSkill.getrank('Mechanical Lore') >= 140
+
+  fput "stow #{GameObj.right_hand.noun}" if GameObj.right_hand.name != 'Empty'
+  fput "stow #{GameObj.left_hand.noun}" if GameObj.left_hand.name != 'Empty'
+  get_from_container('prayer chain')
+  chain_contents = DRC.bput("look at my chain", /^Strung on to the prayer bead chain you see .*\./, 'There are currently no beads on it.')
+  bead_count = DRC.list_to_nouns(chain_contents.match(/^Strung on to the prayer bead chain you see (.*)\./).to_a[1])
+  return unless bead_count.count < 10 
```

Comment:
Would `return if bead_count.count >= 10` be clearer?

Why does the count matter?